spec/generators/scenic/view/view_generator_spec.rb

```diff
@@ -48,4 +48,19 @@
       expect(view_definition).to exist
     end
   end
+
+  if Rails.gem_version >= Gem::Version.new("5.0")
```

Comment:
I'm not convinced we need to keep this test. This test passing when it was written + ongoing passing of the test suite seems sufficient.